csrc/serde/fusion_cache.fbs

```diff
@@ -127,6 +128,7 @@ union RecordData {
   TensorCreationSymbolic,
   Vector,
   Welford,
+  Sort,
```

Comment:
nit: Didn't want to use `Argsort` for the RecordData?

Reply:
I imagine we'll add more sorting-based ops afterwards. like `torch.sort`.
Just trying to keep it general.